app/src/ui/diff/image-diffs/two-up.tsx

```diff
@@ -30,37 +31,52 @@ export class TwoUp extends React.Component<ITwoUpProps, {}> {
+    const previousImageBytes = new Bytes(this.props.previous.bytes)
     const currentImageSize = this.props.currentImageSize || zeroSize
+    const currentImageBytes = new Bytes(this.props.current.bytes)
+    const diffPercent = previousImageBytes.percentDiff(currentImageBytes.bytes)
+    const diffBytes = previousImageBytes.diff(currentImageBytes.bytes)
 
     return (
-      <div className="image-diff-two-up" ref={this.props.onContainerRef}>
-        <div className="image-diff-previous">
-          <div className="image-diff-header">Deleted</div>
-          <DiffImage
-            image={this.props.previous}
-            onElementLoad={this.props.onPreviousImageLoad}
-            style={style}
-          />
+      <div className="image-diff-container" ref={this.props.onContainerRef}>
+        <div className="image-diff-two-up">
+          <div className="image-diff-previous">
+            <div className="image-diff-header">Deleted</div>
+            <DiffImage
+              image={this.props.previous}
+              onElementLoad={this.props.onPreviousImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {previousImageSize.width}px |{' '}
-            <span className="strong">H:</span> {previousImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {previousImageSize.width}px |{' '}
+              <span className="strong">H:</span> {previousImageSize.height}px |{' '}
+              <span className="strong">Size:</span>{' '}
+              {previousImageBytes.format()}
+            </div>
           </div>
-        </div>
 
-        <div className="image-diff-current">
-          <div className="image-diff-header">Added</div>
-          <DiffImage
-            image={this.props.current}
-            onElementLoad={this.props.onCurrentImageLoad}
-            style={style}
-          />
+          <div className="image-diff-current">
+            <div className="image-diff-header">Added</div>
+            <DiffImage
+              image={this.props.current}
+              onElementLoad={this.props.onCurrentImageLoad}
+              style={style}
+            />
 
-          <div className="image-diff-footer">
-            <span className="strong">W:</span> {currentImageSize.width}px |{' '}
-            <span className="strong">H:</span> {currentImageSize.height}px
+            <div className="image-diff-footer">
+              <span className="strong">W:</span> {currentImageSize.width}px |{' '}
+              <span className="strong">H:</span> {currentImageSize.height}px |{' '}
+              <span className="strong">Size:</span> {currentImageBytes.format()}
+            </div>
           </div>
         </div>
+        <div className="image-diff-summary">
+          Diff:{' '}
+          <span className={diffBytes.bytes > 0 ? 'added' : 'removed'}>
```

Comment:
What if there is no size difference?

Reply:
@iAmWillShepherd Good point, I've added some logic for this

![no size diff](https://user-images.githubusercontent.com/988790/39054673-8ae0d452-44b2-11e8-8fcf-578a0980da8b.png)